install/resources/upgrade12.php

```diff
@@ -1280,7 +1280,7 @@ function upgrade12_dbchanges5()
 		$view_count++;
 	}
 
-	$avatardimensions = str_replace('x', '|', my_strtolower($mybb->settings['postmaxavatarsize']));
+	//$avatardimensions = str_replace('x', '|', my_strtolower($mybb->settings['postmaxavatarsize']));
```

Comment:
Wouldn't be better to strip this out completely? :)

Reply:
There is similar issue with probably upgrade2.php as well, which I did skipped.

Reply:
Done.